redfish-core/lib/fan.hpp

```diff
@@ -405,82 +405,77 @@ inline void getFanSpeed(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             << chassisId << " fan = " << fanId;
         return;
     }
-    else
-    {
-        const std::array<std::string, 1> sensorInterfaces = {
-            "xyz.openbmc_project.Sensor.Value"};
-        crow::connections::systemBus->async_method_call(
-            [asyncResp, fanId,
-             chassisId](const boost::system::error_code ec,
-                        const std::vector<std::string>& sensorpaths) {
-                if (ec)
+    const std::array<std::string, 1> sensorInterfaces = {
```

Comment:
What happened to the else here?

Reply:
clang-tidy wants it gone since the if has a return

Reply:
```
if (day == "saturday")
{
   std::court << "Saturday!";
   return;
}
else 
{
    std::court << "Not Saturday";
    return;
}
```    
is the same as 
```
if (day == "saturday")
{
     std::court << "Saturday!";
     return;
}
std::court << "Not Saturday";
return;
```